src/views/ManageList.tsx

```diff
@@ -153,6 +199,24 @@ export function ManageList({ listPath }: Props) {
 						</button>
 					</form>
 					<Toaster />
+					<form onSubmit={(e) => handleInvite(e, listPath)}>
+						<label htmlFor="receipient-email">
+							<input
+								id="receipient-email"
+								type="email"
+								name="receipient-email"
+								value={emailName}
+								onChange={handleEmailNameChange}
+								required
+								aria-label="Enter the user name to share list"
+								aria-required
+							/>
+						</label>
+						<button type="submit" aria-label="Add item to shopping list">
+							Invite User
+						</button>
+					</form>
+					<Toaster />
```

Comment:
note: I don't think there needs to be 2 `<Toaster />` elements in this. It should only need one.

Reply:
So cool! Maha and I were wondering that! We figured one form === one toaster element.

Reply:
Yeah! We really were curious about that. I thought it was a <Toaster/> per form as well. After looking into it some more, its good practice to "import { Toaster } from 'react-hot-toast' "  into the root of the application , probably in our App.tsx file. By doing that,  we will only need to  " import { toast } from 'react-hot-toast' " into any of our needed files, but we wont need to place any <Toaster/> elements since all our app components will have access to it :)